enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/compute/ComputeInstanceItem.java

```diff
@@ -18,21 +18,75 @@
  */
 package org.netbeans.modules.cloud.oracle.compute;
 
+import com.oracle.bmc.core.ComputeClient;
+import com.oracle.bmc.core.VirtualNetworkClient;
+import com.oracle.bmc.core.model.Vnic;
+import com.oracle.bmc.core.model.VnicAttachment;
+import com.oracle.bmc.core.requests.GetVnicRequest;
+import com.oracle.bmc.core.requests.ListVnicAttachmentsRequest;
+import com.oracle.bmc.core.responses.GetVnicResponse;
+import com.oracle.bmc.core.responses.ListVnicAttachmentsResponse;
+import java.util.List;
+import org.netbeans.modules.cloud.oracle.OCIManager;
 import org.netbeans.modules.cloud.oracle.items.OCID;
 import org.netbeans.modules.cloud.oracle.items.OCIItem;
+import org.openide.util.RequestProcessor;
 
 /**
  *
  * @author Jan Horvath
  */
 public final class ComputeInstanceItem extends OCIItem {
-
+    private String publicIp = null;
+    private final RequestProcessor RP = new RequestProcessor();
```

Comment:
Rather use `new RequestProcessor(ComputeInstanceItem.class)` so it is is annotated on thread dump (will set the name of the daemon thread).